### ISSUE TITLE
Guru is assigned CLASS_MISSIONARY

### ISSUE THREAD

**Issue by Infixo** [Infixo/Civ6-Real-Fixes#25]
Why? There is a separate CLASS_GURU for it.

**Comment by whorble** [comment]
Related: Gurus are not present in the religious golden age modifiers. Should they be? Use a single religious class req instead of 3 explicit unit reqs? (in case mods add other religious units)

**Comment by Infixo** [comment]
> Related: Gurus are not present in the religious golden age modifiers. Should they be? Use a single religious class req instead of 3 explicit unit reqs? (in case mods add other religious units)

Do you mean Exodus of the Evangelists? If so, then it is ok imo. The description is precise and matches how it works. No issues here.
Besides, Gurus cannot spread religion and I think this dedication is about spreading the religion passively (+2 charges).

**Comment by whorble** [comment]
Yes, the +2 charges maybe not, but the +2 movement I think would be appropriate, so they can keep up with the other units.
